php_apcu/assets/configuration/spec.yaml

```
@@ -0,0 +1,26 @@
+name: php_apcu
+files:
+- name: php_apcu.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - name: url
+      required: true
+      description: |
+        Provide a URL to the collection program.
+        APCu does not have a metric output by itself.
+        * DD_ROOT/embedded/lib/python3.8/site-packages/datadog_checks/php_apcu/assets/exporter/apcu-dd-handler.php
+        It will work if you refer to it as a php file, but you may want to configure it as follows.
+        ```
+        Alias /apcu-status /path/to/apcu-dd-handler.php
+        <Location /apcu-status>
+            Require all denied
+            Require local
+        </Location>
+        ```
+      value:
+        type: string
+        example: http://localhost/apcu-status
```

```suggestion
        example: http://localhost/apcu-status
     - template: instances/http
     - template: instances/default
```
Missing default template for instance level.

If you implement the request wrapper as mentioned before, you can take advantage of the `http` config template.

---

same fixed 7277ab326a70898fad0201f53ef010d6298e3d25